app/controllers/users_controller.rb

```diff
@@ -7,7 +7,13 @@ class UsersController < ApplicationController
   before_action :admin_user, only: :destroy
 
   def index
-    @users = User.paginate page: params[:page], per_page: Settings.per_page
+    @users = User.where(activated: true)
```

Comment:
tìm hiểu về `scope in rails` và viết scope cho code `.where(activated: true)` nhé